src/utils/object.ts

```diff
@@ -53,3 +53,23 @@ export function startsWith(target: string, searchString: string, position?: numb
 
   return target.indexOf(searchString, effectivePosition) === effectivePosition;
 }
+
+/**
+ * Returns true if the searchString sequence is the same as the corresponding target sequence
+ * starting at the given target position (defaulting to 0); otherwise returns false.
```

Comment:
Similar to updating the `@param` doc for `position`, it may make sense to update the reference here from "starting" to "ending" position as well as the mention of the default value of `0` to the default of the given `target` string length.

Reply:
updated the comment to accurately reflect the logic behind the function!